scripts/manage_portals.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import json
+import sys
+
+from azul.portal_service import PortalService
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(
+        description='Upload/download portal integrations database with safe concurrency.'
+    )
+    parser.add_argument('action', choices=('GET', 'PUT'), help='Whether to download (GET) or upload (PUT).')
```

Comment:
Apologies, I meant to ask for this in my previous review: Can we make the usage `manage_portals.py --get | --put [FILE | -]`? I know mandatory options are a contradiction but it just feels more natural to me in this case.

https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group might come in handy for this.